recipe/build.sh

```diff
@@ -18,7 +18,6 @@ export CPPFLAGS="${CPPFLAGS} -DSQLITE_ENABLE_COLUMN_METADATA=1 \
                              -DSQLITE_MAX_VARIABLE_NUMBER=250000 \
                              -DSQLITE_MAX_EXPR_DEPTH=10000 \
                              -DSQLITE_ENABLE_GEOPOLY \
-                             -DSQLITE_ENABLE_JSON1 \
```

Comment:
Do these flags not work anymore? If they do, maybe we should just keep them, to be explicit.

Reply:
The removed flag is a no-op. It was replaced with a flag to disable JSON1